clang/include/clang/Basic/TargetBuiltins.h

```diff
@@ -152,12 +152,22 @@ namespace clang {
   };
   }
 
+  namespace RISCVXCV {
+  enum {
+    LastRVVBuiltin = RISCVVector::FirstTSBuiltin - 1,
+#define BUILTIN(ID, TYPE, ATTRS) BI__builtin_riscv_cv_##ID,
+#include "clang/Basic/BuiltinsRISCVXCV.def"
```

Comment:
Can we include BuiltinsRISCVXCV.def from BuiltinsRISCV.def and not create a new RISCVXCV namespace? RVV it separated because we use FirstRVVBuiltin and LastRVVBuiltin to catch all RVV builtins for some code.

Reply:
There is no BuiltinsRISCV.def and I can't find a way to make it work like BuiltinRISCVVector.def where rvv and sifive builtins can be in separate files (without modifying BuiltinsRISCV.td).  
I have rewrite BuiltinsRISCVXCV.def in the same format as BuiltinsRISCV.td and included it in BuiltinsRISCV.td for now, I'm willing to explore further if there is better solution.

Reply:
I forgot that BuiltinsRISCV.def had been migrated to BuiltinsRISCV.td